app/controllers/users_controller.rb

```diff
@@ -21,6 +21,10 @@ def edit
 		render partial:'form'
 	end
 
+	def allnames
+		render json: User.all.map {|model| model.tagName }
```

Comment:
also, I'm not entirely sure of performance characteristics but I think there may be a better way than using `map` here to just pull out all the tagNames from the User